### PR TITLE
Webumenia 1792 shuffle front end

### PR DESCRIPTION
**Pull request by eronisko**
# Description

image
image

----
**ℹ You can test this either at https://test.webumenia.sk/, or you'll have to connect to the IIP server via SSH tunnel**
In your .env:
```
IIP_PRIVATE=http://127.0.0.1:8002
```
Then create the tunnel
```
$ ssh -N -L 8002:127.0.0.1:8002 webumenia.sk
```

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: resources/js/components/home/ShuffleOrchestrator.vue [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<script>
+// Shuffles an array
+// https://stackoverflow.com/a/2450976
+function shuffle(inputArray) {
```
Comment (igor-kamil) [inline resources/js/components/home/ShuffleOrchestrator.vue:4]: nice!

File: resources/js/components/home/ShuffleOrchestrator.vue [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+                const nextItem = this.shuffledItems[this.nextItemIndex]
+                const nextFilter =
+                    nextItem.filters[Math.floor(Math.random() * (nextItem.filters.length - 1))]
+
+                // Start shuffling filters
+                const numberOfShuffles = 4
+                const shuffledFilters = shuffle(this.items.flatMap((si) => si.filters))
+
+                await sleep(300)
```
Comment (igor-kamil) [inline resources/js/components/home/ShuffleOrchestrator.vue:83]: [dust] you know I am not the fan of absolute numbers...  :D

**Review by igor-kamil — APPROVED** [review]
nice ✨

File: resources/js/components/home/ShuffleOrchestrator.vue (hunk 2 above)
Comment (eronisko) [reply resources/js/components/home/ShuffleOrchestrator.vue:83]: Yes I remember :) I considered this to be OK in the "animation" world, e.g. https://tailwindcss.com/docs/transition-duration

**Pull request merged**